Source/WebKit/UIProcess/API/Swift/View+WebViewModifiers.swift

```diff
@@ -63,6 +67,16 @@ extension View {
     public func webViewAllowsElementFullscreen(_ value: Bool = true) -> some View {
         environment(\.webViewAllowsElementFullscreen, value)
     }
+
+    @_spi(Private)
+    public func webViewCameraCaptureState(_ value: Binding<WKMediaCaptureState>) -> some View {
```

Comment:
I think I’m missing what ensures that the bindings are updated promptly for their initial layout pass. While I understand using a view modifier for these, since they’re very UI-bound, this state still suffers from the denormalized ownership problem that we avoided by placing stuff like `title` directly on `WebPage`. Basically, by forcing clients to own some canonical `@State` here, while WebKit also has it’s own definition, we set ourselves up to need to ensure these stay in sync. This is doable, but less free than a `var { get set }` on the `WebPage` object would be.

We also still need to work out how “re-parenting” of a `WebPage` will work with this API, which would be a similar synchronization event we’d want to test for. Ex:
```swift
struct Example : View {
  var page: WebPage
  @State var captureState1: WKMediaCaptureState = .none
  @State var captureState2: WKMediaCaptureState = .none

  var body: some View {
    if Bool.random() { // Extremely contrived example
      WebView(page).webViewCameraCaptureState($captureState1)
    } else {
      // Separate view identity here
      WebView(page).webViewCameraCaptureState($captureState2)
    }
  }
}
```